homestead/after.sh

```diff
@@ -7,21 +7,44 @@
 # If you have user-specific configurations you would like
 # to apply, you may also create user-customizations.sh,
 # which will be run after this script.
-
+echo "########\n# Update APT repositories and install development requirements from the apt repos\n######\n"
+# Start by updating the apt dependencies making the base box up to date
 sudo apt update
-sudo apt upgrade -y
+# Install the plantuml dependency (used to generate UML diagrams in the markdown files
 sudo apt install plantuml -y
 
-cd /home/vagrant/code
-
+echo "\n########\n# Update PHP settings\n######\n"
+# Use our onw PHP.ini
 sudo cp /home/vagrant/code/homestead/php.ini /etc/php/7.2/mods-available/custom.ini
+
+# Use PHP 7.2 by default in our environment and enable Xdebug
 sudo phpenmod -v 7.2 custom
-COMPOSER_MEMORY_LIMIT=-1 composer install
+php72
 xon
```

Comment:
In my project this section reads:

```
sudo phpenmod -v 7.2 custom
xon xdebug
```

I think it's important to add the xdebug after the xon statement

Reply:
I don't think so as `xon` is an alias for `sudo phpenmod -s cli xdebug`
`homestead/vendor/laravel/homestead/resources/aliases:14`